nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -442,7 +442,8 @@ in
 
     virtualisation.pathsInNixDB = [ config.system.build.toplevel ];
 
-    virtualisation.qemu.options = [ "-vga std" "-usbdevice tablet" ];
+    virtualisation.qemu.options = [ "-machine gic-version=host" "-cpu host"
+                                   "-machine virt" "-device virtio-gpu" ];
```

Comment:
I'm not sure these are useful, I don't think they ever worked. I plan on removing these changes.